main/src/http/http_service.rs

```diff
@@ -442,10 +455,15 @@ impl HttpService {
                         .await
                         .map_err(|e| e.into());
 
-                    let (tenant, db, user) =
-                        (ctx.tenant(), ctx.database(), ctx.user_info().desc().name());
+                    let (tenant, db, user, addr) = (
+                        ctx.tenant(),
+                        ctx.database(),
+                        ctx.user_info().desc().name(),
+                        addr.as_str(),
+                    );
 
-                    metrics.writes_inc(tenant, user, db, addr.as_str());
+                    metrics.writes_inc(tenant, user, db, addr);
+                    metrics.write_data_in_inc(tenant, user, db, addr, points_len);
```

Comment:
```suggestion
                    metrics.write_data_in_inc(tenant, user, db, addr, req.len());
```

points_len is the serialized data size of flatbuffer, suggest to use the original data len directly